config/default.cfg

```diff
@@ -62,7 +62,7 @@ cfg$model_name <- "REMIND"
 cfg$validationmodel_name <- "VALIDATIONREMIND"
 
 #### model version of the overall model (used for run statistics only).
-# Use extension "-rc" for release candidate and "-dev" for developer version
+# automatically generated for development versions, updated by hand for releases
 cfg$model_version <- "3.2.2-dev"
```

Comment:
we need to start from 3.2.1, but this will happen automatically based on the master branch with the first merge after this PR is merged?